docs/developing-fuzzbench/adding_a_new_benchmark.md

```diff
@@ -60,9 +60,10 @@ PYTHONPATH=. python3 benchmarks/oss_fuzz_benchmark_integration.py -p bloaty
 The script should create the benchmark directory in
 `benchmarks/$PROJECT_$FUZZ_TARGET` (unless you specify the name manually) with
 all the files needed to build the benchmark. You may need to remove unnecessary files
```

Comment:
s/may need/should ?
Also, is removing *.options optional/recommended or required. if recommended, then need to phrase it that way ? E.g. "*.options is unused and we recommend removing them as well"